docs/docs/external/getting-started/enabling-weaver-network/fabric.md

```diff
@@ -0,0 +1,398 @@
+- **Verification policies**:
+  Taking the same example as above, an example of a verification policy for a B/L requested by the `trade-finance-network` from the `trade-logistics-network` is as follows:
+  ```
+  {
+      "securityDomain":"trade-logistics-network",
+      "identifiers":
+          [
+              {
+                  "pattern":"tradelogisticschannel:shipmentcc:GetBillOfLading:*",
+                  "policy":
+                      {
+                          "type":"Signature",
+                          "criteria":
+                              [
+                                  "ExporterMSP",
+                                  "CarrierMSP"
+                              ]
+                      }
+              }
+          ]
+  }
+  ```
+  In this sample, a single verification policy rule is specified for data views coming from `trade-logistics-network`: it states that the data returned by the `GetBillOfLading` query made to the `shipmentcc` chaincode on the `tradelogisticschannel` channel requires as proof two signatures, one from a peer in the organization whose MSP ID is `ExporterMSP` and another from a peer in the organization whose MSP ID is `CarrierMSP`.
+
+  You need to record this policy rule on your Fabric network's channel by invoking either the `CreateVerificationPolicy` function or the `UpdateVerificationPolicy` function on the Fabric Interoperation Chaincode that is already installed on that channel; use the former if you are recording a set of rules for the given `securityDomain` for the first time and the latter to overwrite a set of rules recorded earlier. In either case, the chaincode function will take a single argument, which is the policy in the form of a JSON string (make sure you escape the double quotes before sending the request to avoid parsing errors). As with the access control policy, you can do this in one of two ways: (1) writing a small piece of code in Layer-2 that invokes the contract using the Fabric SDK Gateway API, or (2) running a `peer chaincode invoke` command from within a Docker container built on the `hyperledger/fabric-tools` image. Either approach should be familiar to a Fabric practitioner.
```

Comment:
Can we mention here that, AccessControlPolicy and Membership needs to added to Source Network (Trade Logisitics Network), but Verification Policy needs to be added to Destination Network (Requesting network: Trade finance network).

Reply:
Fixed